### PR TITLE
Queue rebuilt

### PR DESCRIPTION
**Pull request by anpolimus**
This pull request covers next tasks:
1. Users scan from d.org has moved from batch into Drupal.queue.
2. Fixed bug with duplicating of the users during rescan.
3. Fixed bug with duplicating of field values during rescan.
4. Added statistics widgets:
    - users_by_year.inc statistics of activity grouped by the year of registration, count of user registrations by year.
    - top_mentors.inc shows top users that are mostly mentioned as mentors  by another Ukrainian users.
   -  members_by_country.inc - statistics of individual Drupal Association members, groupped by country.

5. Added display of count of individual members at ukrainian community.

6. Fixed bug when project theme wasnt installed during reinstall process.

### CODE REVIEW

File: docroot/.gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@ sites/*/settings*.php
 # Ignore paths that contain user-generated content.
 sites/*/files
 sites/*/private
+*.sass-cache
```
Comment (BR0kEN-) [inline docroot/.gitignore]: I guess we can just add the `.*` entry.

Comment (anpolimus) [inline docroot/.gitignore]: Please, explain detaily what are talking about?

Comment (BR0kEN-) [inline docroot/.gitignore]: I meant that we don't need add files/folders, which names starts from a dot, to .gitignore every time.

Comment (anpolimus) [inline docroot/.gitignore]: Ah, thx.

**Pull request merged**